data/certChooser.html

```diff
@@ -16,7 +16,7 @@
   </div>
   <p>To confirm you agree to sign this text message using your selected certificate, please confirm by entering the master password:</p>
   <input id="certPassword" type="password">
-  <button onclick="doCancel()">Cancel</button><button onclick="doOK()">OK</button>
+  <span class="buttonBox"><button id="btnCancel" onclick="doCancel()">Cancel</button><button id="btnOK" onclick="doOK()">OK</button></span>
```

Comment:
If we don't need to refer to the OK button directly, it doesn't seem like we need to add an id for it.